### ISSUE TITLE
Hand off audit to OCADU

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#1341]
## Description
As a Notify team member, I need to be able to coordinate an external a11y audit so that I can keep the list of known a11y issues up to date. 

WHY are we building?
- We get to do an external audit with OCADU
- It needs to happen now in September

WHAT are we building?
- Scope and access for the auditors. Most of it is already outlined in the requirement doc below, we just need to hand off to auditors and make sure they have everything they need. 

VALUE created by our solution
- Refreshed list of known a11y issues

WHEN are we building
- We need to hand off for the external audit before the end of this sprint (September 24) 

## Documentation and Artifacts
- [audit requirement](https://docs.google.com/document/d/1JvV__6cz-BpbBkHKXC871_m9WJNeQZLS1MODCD0u_eo/edit)
- [audit resources and alignment](https://docs.google.com/document/d/1B8zyJhrqfkR-pESZBPFzDrZNsedQE62EZwdIf8BY95E/edit)
- [Figma diagrams for audit](https://docs.google.com/document/d/1B8zyJhrqfkR-pESZBPFzDrZNsedQE62EZwdIf8BY95E/edit)

## Acceptance Criteria
Given some context, when (X) action occurs, then (Y) outcome is achieved 

- [x] Audit is scoped
- [x] Access is sorted out for auditor on staging. 
- [x] Notify meets with auditors to hand off

**Comment by foudamo** [comment]
Slack Thread: https://gcdigital.slack.com/archives/C04TQMJCXEW/p1693406095677119

**Comment by foudamo** [comment]
Goals of kick off meeting (with Lisa Liskovoi lliskovoi@ocadu.ca): 
- Kickoff the project. Say hello, introduce ourselves. 
- Ask OCADU: if there's training? 
- Clarify on the dates/deadlines. Now-Oct 1st. Correct? Is that audit, or do we also get feedback in that time? 
- Go over the document, focus & prioritize what's on the document

**Comment by foudamo** [comment]
cc @amazingphilippe 
1. Notify: to make a map of the 13-14 tasks and share it with Lisa and co. 
2. Notify: to prioritize the tasks as more important and less important 
3. Notify: find out what kind of user tests data/information can we share with Lisa and co. Lisa is looking to know what kind of areas have we researched, what are missing, and what design problems we have uncovered in our testing
4. Lisa: based on task 1 & 2, estimate how many hours OCAD can dedicate, and what realistically can we have audited

**Comment by yaelberger-commits** [comment]
Meeting with Lisa last week, working through action items this week

**Comment by yaelberger-commits** [comment]
Phil looking at IA diagram and creating scenarios for Lisa

**Comment by amazingphilippe** [comment]
Documented key handoff resources for Lisa in [this document](https://docs.google.com/document/d/1B8zyJhrqfkR-pESZBPFzDrZNsedQE62EZwdIf8BY95E/edit)

**Comment by amazingphilippe** [comment]
Sent documentation to Lisa and scheduling a meeting to set up the email transfers soon.

**Comment by adriannelee** [comment]
Meeting planned with Lisa tomorrow to set up the email re-directs. Gave her access to the a11y tracker on github.

**Comment by amazingphilippe** [comment]
Lisa and Phil met. Email transfer is set up. Currently just on stand-by if Lisa needs any assistance. Moving to blocked while this work is with OCADU

**Comment by mtoutloff** [comment]
Semi-regularly, emails about audit popping up in inbox
